.ruby-version

```diff
@@ -1 +1 @@
-2.2.0
```

Comment:
How did you choose this version? Just curious.

I will add installing new ruby to the deploy checklist for 7.3.

Reply:
Honestly, it was the one I already had installed and ruby-build takes forever on the macbook air. In a vacuum I'd suggest we go all the way to 2.4.3.

Reply:
I think that is best if you don't mind. Run it the bg while you work on sth else? The fewer times we have to update ruby on the farm of ELMOs out there, the better.

Reply:
Already running in the background now. There's probably a way to make capistrano run `rbenv install` for us, do you want me to look into adding that to the deploy project?